src/gjk/power_to_energy.py

```diff
@@ -0,0 +1,173 @@
+from gjk.models import ReadingSql
+from gjk.models import DataChannelSql
+from gjk.config import Settings
+from sqlalchemy.orm import Session, sessionmaker
+from sqlalchemy.orm import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy import asc 
+from sqlalchemy import Column, BigInteger, Integer, String, Float
+from sqlalchemy import UniqueConstraint
+import pendulum
+import dotenv
+from datetime import datetime
+import pandas as pd
```

Comment:
This is a new package for the repo. It needs to be added to the pyproject.toml  (poetry add pandas)